deck/models.py

```diff
@@ -210,8 +213,7 @@ def user_already_voted(self, user):
 
     def user_can_vote(self, user):
         can_vote = False
-        if self.user_already_voted(user) or \
-           (self.author == user and not self.event.author == user):
+        if self.author == user and not self.event.author == user:
```

Comment:
I know this code was already here but it is very weird because it falls into a `pass`.